server/routes/users.js

```diff
@@ -14,8 +16,36 @@ router.get('/', function(req, res) {
   });
 });
 
+router.post('/login',
+  passport.authenticate('local'),
+  function(req, res) {
+    res.send({ user: req.user });
+  }
+);
+
+router.delete('/logout', function(req, res) {
+  req.logout();
+  res.send(true);
+});
+
+router.post('/register', function(req, res) {
+  var body = req.body;
+  var salt = bcrypt.genSaltSync();
+  var attributes = {
+    email: body.email,
+    name: body.name,
+    encryptedPassword: bcrypt.hashSync(body.password, salt),
+    passwordSalt: salt
+  };
+  User.create(attributes).then(function(user) {
+    res.status(201).send({ user: user });
```

Comment:
@Vakrim I think it would be nice to have ```user.toJson()``` method like it is in snippets to identically serialize objects across application